support.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Support
+page_title: Support
+description: Copyright
+slug: support-page
+tags: Fiddler support, trial, pro version, paid support
+published: True
+position: 80
+---
+
+## Fiddler Everywhere Support 
+
+ 
+#### Who has access to support? 
+
+All users who have obtained a Trial subscription or PRO plan subscription will have access to support for the duration of the active subscription period, the table below describes the comparison between different plans and level of support offered for each plan. 
+
+
+<!-- TODO the plan table here -->
+|   |**Free**   |**Trial**   |**Pro**  |
+|---|---|---|---|
+| Email support |  - |  __Unlimited__* | __Unlimited__* |
+| Community Forums   | Yes | Yes | Yes |
+| Feedback Portal   | Yes | Yes | Yes |
+
+*subject to __fair usage policy__
+
+#### What is the Fair Usage Policy? 
+
+Most of our product licenses include unlimited access to the support service. In order to keep providing our industry-leading level of support, we have instituted a fair usage policy for our support services: 
+
+A support manager will review customer activities that exceed at least ten times the average amount of support requests to determine if such activities are deemed to be excessive. 
+
+#### How to create a support ticket? 
```

Comment:
Consider rephrasing to: "**How to contact support?**" because currently those are just emails, not tickets.